docs/02_cleos/03_command-reference/set/set-account-permission.md

```diff
@@ -42,7 +42,8 @@ The following information shows the different positionals and options you can us
 
 ## Requirements
 * Install the currently supported version of `cleos`.
-[[info | Note]] | `Cleos` is bundled with the EOSIO software. [Installing EOSIO](../../00_install/index.md) will also install the `cleos` and `keosd` comand line tools.  
+[[info | Note]] 
+| `Cleos` is bundled with the EOSIO software. [Installing EOSIO](../../../00_install/index.md) will also install the `cleos` and `keosd` comand line tools.  
```

Comment:
`cleos` is a program name. @bobgt Is it okay for system names (which we wrap in backticks as a convention) to use sentence case (as in `Cleos`) even when starting a sentence?